backend/src/main/java/de/neuefische/backend/service/RandomIdService.java

```diff
@@ -0,0 +1,10 @@
+package de.neuefische.backend.service;
+
+import java.util.UUID;
+
```

Comment:
i think there's a @Service annotation missing

Reply:
Thanks for the check.
I added the missing Annotation :-)